covid_moonshot/analysis/plots.py

```diff
@@ -252,6 +253,40 @@ def plot_convergence(
     return fig
 
 
+def plot_poor_convergence_fe_table(
+    runs: List[int], runs_details: List[int], energy_cutoff: float = 1.0,
+) -> plt.Figure:
```

Comment:
Would you mind adding a docstring with a brief description of the output and arguments (with units)?